lib/Segments/DynamicSegments/Filters/WooCommerceTotalSpent.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace MailPoet\Segments\DynamicSegments\Filters;
+
+use MailPoet\Entities\DynamicSegmentFilterEntity;
+use MailPoet\Entities\SubscriberEntity;
+use MailPoetVendor\Carbon\Carbon;
+use MailPoetVendor\Doctrine\DBAL\Query\QueryBuilder;
+use MailPoetVendor\Doctrine\ORM\EntityManager;
+
+class WooCommerceTotalSpent implements Filter {
```

Comment:
Should we create a test for this class? Similar to `WooCommerceNumberOfOrdersTest`? I am not sure about that so that is why I am asking you about your opinion.

Reply:
Yes, it's possible. I will have to add some metadata about order value. Good idea.